mrmustard/lab/abstract/state.py

```diff
@@ -498,6 +515,17 @@ def __getitem__(self, item) -> State:
         self._modes = item
         return self
 
+    def bargmann(self) -> Optional[tuple[ComplexMatrix, ComplexVector, complex]]:
+        r"""Returns the Bargmann representation of the state."""
+        if self.is_gaussian:
+            if self.is_pure:
+                A, B, C = bargmann.wigner_to_bargmann_psi(self.cov, self.means)
```

Comment:
May change from wiger to husimi to bargmann?

Reply:
`wigner_to_bargmann_psi` calls `wigner_to_husimi` as a step. Is that what you meant?